guides/_subsections/add-and-edit-recording-metadata-labels.md

```diff
@@ -10,72 +10,72 @@ Users can attach metadata or custom labels to JDK flight recordings managed by C
+      The labels section will appear as a form where you can <i>Add</i>, <i>Edit</i>, or <i>Delete</i> existing labels, just like before when we created the <code>Recording</code>. Fix the typo, and click <i>Save</i> to save your edited labels.
     </p>
     {% endcapture %}
     {% include howto_step.html
-      summary="Edit an existing label"
+      summary="Edit an Existing Label"
       image-name="2.3.0/add-and-edit-recording-metadata-labels-3.png"
       text=edit-an-existing-label-text
     %}
   </li>
   <li>
     {% include howto_step.html
-      summary="View your edited labels"
+      summary="View your Edited Labels"
       image-name="2.3.0/add-and-edit-recording-metadata-labels-5.png"
       text="
-        The recording labels should be updated in the Active Recordings table. 
+        The <code>Recording</code> labels should be updated in the <i>Active Recordings</i> table.
       "
     %}
   </li>
   <li>
     {% include howto_step.html
-      summary="<i>(Optional)</i> Archive your recording to view labels copied onto the archived recording"
+      summary="<i>(Optional)</i> Archive your <code>Recording</code> to view labels copied onto the <code>Archived Recording</code>"
       image-name="2.3.0/add-and-edit-recording-metadata-labels-6.png"
       text="
-        On the Active Recordings table, click the checkbox next to the recording, then click <i>Archive</i> to archive your recording. Notice that the archived recording also copies the labels from the active recording. You can also add labels to any recording uploaded to Cryostat’s archives.
+        On the <i>Active Recordings</i> table, click the <i>Checkbox</i> next to the <code>Recording</code>, then click <i>Archive</i> to archive your <code>Recording</code>. Notice that the <code>Archived Recording</code> also copies the labels from the <code>Active Recording</code>. You can also add labels to any <code>Recording</code> uploaded to <b>Cryostat’s</b> archives.
       "
     %}
   </li>
   <li>
     {% capture bulk-edit-recording-labels-text %}
     <p>
-      Create another recording on the same target. Then select both recordings on the Recordings Table and click <i>Edit Labels</i> and <i>Edit</i>.This time, only labels that are present on both recordings will be shown in the form. Let's delete the two common template-related labels, and add a new label to both recordings. Then finally, click <i>Save</i>.
+      Create another <code>Recording</code> on the same target. Then select both <code>Recordings</code> on the <i>Recordings Table</i> and click <i>Edit Labels</i> and <i>Edit</i>.This time, only labels that are present on both <code>Recordings</code> will be shown in the form. Let's <i>Delete</i> the two common template-related labels, and add a new label to both <code>Recordings</code>. Then finally, click <i>Save</i>.
```

Comment:
"... and <i>Edit</i>. This time, ..... " there should be a space after the period :)

Reply:
also "Delete" is capitalized and italicized, but add isn't? I'm pretty sure there isn't a "delete" button (just an "x" button), but there is an "Add Label"  buttons